jobs/buildhub/inventory_to_records.py

```diff
@@ -244,10 +244,14 @@
     for f in files:
         filename = f['name']
         if is_release_build_metadata(product, version, filename):
-            metadata = await fetch_json(session, url + filename)
-            metadata['buildnumber'] = build_number
-            _release_metadata[url] = metadata
-            return metadata
+            try:
+                metadata = await fetch_json(session, url + filename)
+                metadata['buildnumber'] = build_number
+                _release_metadata[url] = metadata
+                return metadata
+            except aiohttp.ClientError as e:
```

Comment:
Isn't it supposed to be a ValueError?

Reply:
No it is an internal exception of `response.json()` from aiohttp